gocontext.go

```diff
@@ -7,23 +7,17 @@ type contextKey struct{}
 var activeSpanKey = contextKey{}
 
 // ContextWithSpan returns a new `context.Context` that holds a reference to
-// the given `Span`.
+// the given `Span`'s `SpanContext`.
 func ContextWithSpan(ctx context.Context, span Span) context.Context {
 	return context.WithValue(ctx, activeSpanKey, span)
 }
 
-// BackgroundContextWithSpan is a convenience wrapper around
-// `ContextWithSpan(context.BackgroundContext(), ...)`.
-func BackgroundContextWithSpan(span Span) context.Context {
```

Comment:
I got rid of this just because I regret it :)

Not related to the change here.